README.md

```diff
@@ -15,17 +15,48 @@ This will ensure that the `PYTHONPATH` will be set correctly.
 
 ## Launch experiments
 
-Open and modify `exp_configs.py` and `launch_command.py` to your needs. They are
-located in `agentlab/experiments/`.
+We provide default settings to run our agents on a few benchmarks. Those can be ran from `launch_exp.py`,
+with the following flags:
+
+```bash
+  -h, --help            show this help message and exit
+  --exp_root EXP_ROOT   folder where experiments will be saved
+  --n_jobs N_JOBS       number of parallel jobs
+  --exp_group_name EXP_GROUP_NAME
+                        Name of the experiment group to launch as defined in exp_configs.py
+  --benchmark {miniwob,workarena.l1,workarena.l2,workarena.l3}
+                        Benchmark to launch
+  --model_name {gpt-3.5,gpt-4o,gpt-4o-vision,cheat,custom}
```

Comment:
let's remove cheat since it's just for unit test, perhaps we can add a section to show how to run unit-tests.